deploy/helm/sumologic/conf/setup/setup.sh

```diff
@@ -142,6 +142,15 @@ TF_LOG_PROVIDER=DEBUG terraform apply \
     -var="create_fields=${CREATE_FIELDS}" \
     || { echo "Error during applying Terraform changes"; exit 1; }
 
+# Setup Sumo Logic monitors if enabled
+{{- if .Values.sumologic.setup.monitors.setupEnabled }}
+bash /etc/terraform/monitors.sh
+{{- else }}
+echo "Installation of the Sumo Logic monitors is disabled."
+echo "You can install them manually later with:"
+echo "https://github.com/SumoLogic/terraform-sumologic-sumo-logic-monitor/tree/main/monitor_packages/kubernetes"
```

Comment:
nobody will see this message, as setup pod is being removed just after it finish it jobs (except failures). please add this info to https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/50b1a462b19790110ecafdc33c75cd848c2fdca8/deploy/helm/sumologic/templates/NOTES.txt